src/app/story/directives/basic.component.ts

```diff
@@ -36,16 +36,21 @@ import { HalDoc, ModalService, TabService } from 'ngx-prx-styleguide';
       <hr/>
 
       <prx-fancy-field required label="Audio Files">
-        <prx-select *ngIf="versionTemplateOptions" class="file-select"  placeholder="Select Templates..."
+        
+        <prx-select
+          *ngIf="versionTemplateOptions" class="file-select"  placeholder="Select Templates..."
           [selected]="versionTemplatesSelected" [options]="versionTemplateOptions"
-          (onSelect)="updateVersions($event)"
+          (onSelect)="updateVersions($event)" [maxSelectedItems]="1"
           [class.invalid]="versionsInvalid"
           [class.changed]="versionsChanged"></prx-select>
         <prx-spinner *ngIf="!undeletedVersions"></prx-spinner>
         <publish-upload *ngFor="let v of undeletedVersions" [version]="v" [strict]="strict"></publish-upload>
         <div *ngIf="undeletedVersions?.length === 0" class="fancy-hint">
           Pick at least one version of your audio files to upload for this episode.
         </div>
+        <div *ngIf="undeletedVersions?.length > 0" class="fancy-hint">
```

Comment:
How does one get into this state?  The dropdown won't let me select > 1 in the first place.  (Or maybe this is just a belt-and-suspenders thing).

In any case ... I find it confusing why I can't click the expand-dropdown at all when I have a template selected.  Like there's no UI feedback on that.  Just the "Clear your selection" message.

Reply:
(this is the UX i find non-intuitive)

![](http://cav.is/i/ihm58.gif)

Reply:
I agree. I can work on better feedback for this.